src/context.cpp

```diff
@@ -259,11 +259,11 @@ namespace Sass {
 
     // do not parse same resource twice
     // maybe raise an error in this case
-    if (sheets.count(inc.abs_path)) {
-      free(res.contents); free(res.srcmap);
-      throw std::runtime_error("duplicate resource registered");
-      return;
-    }
+    // if (sheets.count(inc.abs_path)) {
```

Comment:
It might lead to memory leaks and it's also not really defined what should happen since importer could return different content for the same url this way.